draft-ietf-quic-load-balancers.md

```diff
@@ -874,10 +874,12 @@ handle these DCIDs, rests on some assumptions:
 * Incoming short headers do not contain DCIDs that are client-generated.
 * The use of client-generated incoming DCIDs does not persist beyond a few round
 trips in the connection.
-* In this interval, some exposed fields (e.g. UDP address and port, source and
-client-generated destination Connection ID) remain constant for all packets in
-the connection. For example, 0RTT and Initial packets use the same CIDs and
-ports.
+* In this interval, some exposed fields (e.g. UDP address and port, client-
```

Comment:
What interval?

 (I think that the list structure is working against you here.)